python/sdist/amici/de_model_components.py

```diff
@@ -607,6 +607,46 @@ def __init__(
         super().__init__(identifier, name, value)
 
 
+class NoiseParameter(ModelQuantity):
+    """
+    A NoiseParameter is an input variable for the computation of ``sigma`` that can be specified in a data-point
+    specific manner, abbreviated by ``np``. Only used for jax models.
+    """
+
+    def __init__(self, identifier: sp.Symbol, name: str):
+        """
+        Create a new Expression instance.
+
+        :param identifier:
+            unique identifier of the NoiseParameter
+
+        :param name:
+            individual name of the NoiseParameter (does not need to be
+            unique)
+        """
+        super().__init__(identifier, name, 0.0)
+
+
+class ObservableParameter(ModelQuantity):
+    """
+    A NoiseParameter is an input variable for the computation of ``y`` that can be specified in a data-point specific
+    manner, abbreviated by ``op``. Only used for jax models.
+    """
```

Comment:
With respect to https://github.com/AMICI-dev/AMICI/issues/11, we might want to use `ObservableParameter` also for non-jax models at some point, without necessarily being able do specify them in a time-point specific manner. How about adding a `is_timepoint_specific` attribute already? To soon to worry about that?

Reply:
Would follow YAGNI here